activesupport/lib/active_support/duration.rb

```diff
@@ -181,6 +181,10 @@ def years(value) #:nodoc:
       #   ActiveSupport::Duration.build(2716146).parts  # => {:months=>1, :days=>1}
       #
       def build(value)
+        unless value.is_a?(::Numeric)
+          raise TypeError, "can't build an `#{self.name}` from a `#{value.class.name}`"
```

Comment:
Can you remove the backticks from the message please - the other message doesn't have them and core Ruby exception messages don't so I'd rather stick to that pattern (though I accept there are places elsewhere in Rails they are added).

Reply:
@pixeltrix: Changed in https://github.com/rails/rails/pull/37013/commits/94fa3e86285c1f05a8f3bbaa20cbdad0e649323a

Reply:
@pixeltrix: changed in https://github.com/rails/rails/pull/37013/commits/94fa3e86285c1f05a8f3bbaa20cbdad0e649323a

Reply:
@pixeltrix: squashed into https://github.com/rails/rails/pull/37013/commits/b8b7e853832ff65509c1c29d0fbe259b7ab5aded